membrane_curvature/tests/test_mdakit_membcurv.py

```diff
@@ -5,13 +5,15 @@
 # Import package, test suite, and other packages as needed
 import pickle
 
+from numpy.testing._private.utils import assert_array_equal
```

Comment:
In general you should avoid using private paths (typically prefixed with an underscore), as they may change in the future. Can you `from numpy.testing import assert_array_equal)?

Reply:
Sorry, I thought I added that change already in a previous PR. That change wasn't saved.